modules/toast/react/stories/stories_visualTesting.tsx

```diff
@@ -39,7 +39,7 @@ storiesOf('Components|Popups/Toast/React/Visual Testing', module)
       >
         {props => (
           <Toast transformOrigin={null} aria-label="Play" {...props}>
-            Your workbook was successfully processed.
+            Successfully processed.
```

Comment:
This is to prove that when the text is a single line, the link won't wrap up. 👍 

You can maybe add a story that has multiple lines to cover that use case as well

Reply:
will do!
thanks